src/structure/pure/MultiTrigger.js

```diff
@@ -51,54 +51,35 @@ define( ['_common'], function( design ) {
     
     // create the actor
     ret_val += '<div class="actor_container" style="float:left">';
-    var buttonCount = 0;
-    
+
     if( data.button1label )
     {
-      ret_val += '<div class="actor switchUnpressed ';
-      if( data.align ) 
-        ret_val += data.align; 
-      ret_val += '">';
```

Comment:
Because there is no align attribute defined for multitriggers in the XSD, so this is unreachable code.